docker/frontend.dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:bionic
+
+ENV LANG C.UTF-8
+
+# install latest python and nodejs
+RUN apt-get update && \
+    apt-get install -y \
+      nodejs \
+      npm \
+      gettext \
+      git \
+      sudo \
+      build-essential \
+      python3 \
+      python3-dev \
+      python3-pip \
+      libxml2-dev \
+      libxslt1-dev \
+      zlib1g-dev
+
+RUN bash -c "pip3 install pip setuptools --upgrade --force-reinstall"
+RUN bash -c "pip3 install virtualenv"
+RUN bash -c "npm install -g bower"
+
+# RUN adduser --system --shell /bin/bash --home "/repo" rtd
+
+# A volume used to share `pex`/`whl` files and fixtures with docker host
+VOLUME /repo
```

Comment:
This has come up any time docker has been discussed, but mounting like this is bound to result in permission errors, especially for any folks on Linux, which is majority of core team.

Reply:
This is the essence of the whole Docker file though: It builds the output assets directly in your local files. I'm on Linux and it works fine. Without the use of a volume, I can't complete this PR. So might as well close it then.

Reply:
@benjaoming when running this docker image, don't the generated files by the docker image end up belonging to `root` user?

Reply:
@humitos not existing files, but I can see that newly created files are owned by `root`. So it applies to `/static/`.

Reference: https://github.com/moby/moby/issues/3124

I can see these options:

 * Not using volumes and instead listing all the files/dirs to copy out of the container
 * Using `docker compose` and whatever features it has (seems from the above reference that it can do something)
 * Changing the setup to be aware of the appropriate UID of the invoking instance and creating a user for this

Reply:
There are some methods of inheriting the `UID` of user invoking Docker into the container and thus getting around the permission issue.

I haven't come across anything better. I can implement this.